From 16fc764a7afe14164ca7aac516bdecb3cb179270 Mon Sep 17 00:00:00 2001
From: Ahmed Morsi <eng.eramax@gmail.com>
Date: Thu, 4 Jun 2026 10:31:20 +0200
Subject: [PATCH] Nova: local changes for AIDL/build support

Removed HIDL/AIDL Android.bp files not needed for host build.
Added parcelable stubs for missing types and patched AIDL imports.

Patch series for the Nova Android-on-Linux runtime.

Change-Id: Ie3736a451ae4be8f375771f681c0843d3db4629f
---
 bench/inodeop_bench/Android.bp | 23 -----------------
 tests/Android.bp               | 45 ----------------------------------
 2 files changed, 68 deletions(-)
 delete mode 100644 bench/inodeop_bench/Android.bp
 delete mode 100644 tests/Android.bp

diff --git a/bench/inodeop_bench/Android.bp b/bench/inodeop_bench/Android.bp
deleted file mode 100644
index 3d1769e..0000000
--- a/bench/inodeop_bench/Android.bp
+++ /dev/null
@@ -1,23 +0,0 @@
-/*
- * Copyright (C) 2020 The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package {
-    default_applicable_licenses: ["Android-Apache-2.0"],
-}
-
-cc_binary {
-    name: "inodeop_bench",
-    srcs: ["inodeop_bench.cpp"],
-}
diff --git a/tests/Android.bp b/tests/Android.bp
deleted file mode 100644
index 95a9580..0000000
--- a/tests/Android.bp
+++ /dev/null
@@ -1,45 +0,0 @@
-package {
-    default_team: "trendy_team_android_kernel",
-    default_applicable_licenses: ["Android-Apache-2.0"],
-}
-
-cc_test {
-    name: "vold_tests",
-    defaults: [
-        "vold_default_flags",
-        "vold_default_libs",
-    ],
-
-    srcs: [
-        "Utils_test.cpp",
-        "VoldNativeServiceValidation_test.cpp",
-    ],
-    static_libs: ["libvold"],
-    shared_libs: ["libbinder"],
-}
-
-cc_fuzz {
-    name: "vold_native_service_fuzzer",
-    defaults: [
-        "vold_default_flags",
-        "vold_default_libs",
-        "keystore2_use_latest_aidl_ndk_shared",
-        "service_fuzzer_defaults",
-        "fuzzer_disable_leaks",
-    ],
-    static_libs: [
-        "libvold",
-        "android.security.maintenance-ndk",
-        "libkeymint_support",
-    ],
-    header_libs: ["libvold_headers"],
-    srcs: [
-        "VoldFuzzer.cpp",
-    ],
-    corpus: ["vold_native_service_fuzzer_corpus/*"],
-    fuzz_config: {
-        cc: [
-            "maco@google.com",
-        ],
-    },
-}
-- 
2.53.0

